README.md

```diff
@@ -106,10 +104,11 @@ To be able to also use the cds models defined in this plugin the `cds-maven-plug
 </plugin>
 ```
 
-If the cds models needs to be used in the `db` folder the `cds-maven-plugin` needs to be included also in the
-`db` folder of the project.
-This means the `db` folder needs to have a `pom.xml` with the `cds-maven-plugin` included and the `cds-maven-plugin`
-needs to be run.
+If the CDS models needs to be used in the `db` folder, the `cds-maven-plugin` needs to be included also in the `db` folder of the project. There are two different approaches to consume and use the Attachments aspect in a CAP Java application.
+
+### With a Maven artifact
+
+This means the `db` folder needs to have a `db/pom.xml` with the `cds-maven-plugin` included and the `cds-maven-plugin` needs to be run.
```

Comment:
This will also include the Java content in the db module. I think we should avoid this. So which scope should the import have?

`compile` would also include the java artifacts at runtime
`provided` would also exclude the model artifacts at runtime

both scopes don't really capture the intent

Reply:
reworked this section and removed the part, where the db folder shall be a maven module.